src/SFA.DAS.Admin.Aan.Web/Models/ManageEvent/SchoolNameViewModel.cs

```diff
@@ -7,4 +7,6 @@ public class SchoolNameViewModel : ManageEventViewModelBase
     public string? SearchTerm { get; set; }
     public string? Name { get; set; }
     public string? Urn { get; set; }
+    public bool DirectCallFromCheckYourAnswers { get; set; }
```

Comment:
The name does make little bit sense, but I am unable to understand why do you need this on top of HasSeenPreview? 
By convention, name of booleans should be a question.